src/chanakya/services/tool_loader.py

```
@@ -61,31 +63,23 @@ async def load_all_mcp_tools_async(force_reload=False) -> List[BaseTool]:
+    app.logger.info(f"Initializing MCPClient for Chanakya with processed config: {cfg_local}")
     client = MultiServerMCPClient(cfg_local)
     app.logger.info("Loading all MCP tools via client for Chanakya (ASYNCHRONOUSLY)...")
 
     tools: List[BaseTool] = []
     try:
         tools = await client.get_tools()
-        app.logger.info(
-            f"Successfully fetched {len(tools)} tools from MCP client asynchronously."
-        )
+        app.logger.info(f"Successfully fetched {len(tools)} tools from MCP client asynchronously.")
     except Exception as e_gen:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)